index.html

```diff
@@ -12,7 +12,15 @@
 <body>
   <div class="body-container">
     <h2 class="main-title">Front-End Tools</h2>
-    <h2 class="main-title">Back-End Tools</h2>
+    <div class="tech-border">
+      <div class="tech-container">
+        <img class="tech-image" src="./assets/react_icon.svg" alt="react">
+        <span class="tech-title">ReactJS</span>
```

Comment:
`ReactJS` is the tech title so you have to empathize it. I would use something like `<h1/> or <h2/>`